ci/clang-format.sh

```diff
@@ -48,7 +48,7 @@ for file in $(git ls-files | grep '\.\(cpp\|h\)$'); do
   fi
 
   formatted="${file}.formatted"
-  clang-format "$file" > "$formatted"
+  /opt/wasi-sdk/bin/clang-format "$file" > "$formatted"
```

Comment:
I'm a little undecided about this. It might be nice to accept the clang-format program as an environment variable, but we also require wasi-sdk to build, so this path should always exist.